src/core/generate.test.ts

```diff
@@ -134,7 +165,7 @@ describe("generate", () => {
     });
 
     it("should not have any errors", () => {
-      expect(errors.length).toBe(0);
+      expect(errors && errors.length).toBe(0);
```

Comment:
I’m curious about this one, `errors` should be an empty array

Reply:
This is because I was initialising it to undefined, I'll refactor this next commit.